### PR TITLE
Update Microsoft.Azure.Management.ManagedServiceIdentity SDK version

### PR DESCRIPTION
**Pull request by wendyNEU**
## Description

Update version of Microsoft.Azure.Management.ManagedServiceIdentity to 0.11.0
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ManagedServiceIdentity/ManagedServiceIdentity/Az.ManagedServiceIdentity.psd1 [hunk 1]
```diff
@@ -110,7 +110,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = 'Update release with latest service features and serialization fixes'
+        ReleaseNotes = 'Update version of Microsoft.Azure.Management.ManagedServiceIdentity to 0.11.0'
```
Comment (markcowl) [inline src/ManagedServiceIdentity/ManagedServiceIdentity/Az.ManagedServiceIdentity.psd1]: What is the practical impact for the customer of this change?

File: src/ManagedServiceIdentity/ManagedServiceIdentity/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -30,6 +30,6 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
-// [assembly: AssemblyVersion("0.7.1")]
-[assembly: AssemblyVersion("0.7.1")]
-[assembly: AssemblyFileVersion("0.7.1")]
+// [assembly: AssemblyVersion("0.8.0")]
```
Comment (markcowl) [inline src/ManagedServiceIdentity/ManagedServiceIdentity/Properties/AssemblyInfo.cs]: You don't need to update this

File: src/ManagedServiceIdentity/ManagedServiceIdentity.Test/ScenarioTests/CrudTests.ps1 [hunk 3]
```diff
@@ -62,6 +62,7 @@ function Test-CrudUserAssignedIdentity
 
         #Delete Identity1
         Remove-AzUserAssignedIdentity -ResourceGroupName $rgName1 -Name $identityName1 -Force;
+        Start-Sleep -s 5
```
Comment (markcowl) [inline src/ManagedServiceIdentity/ManagedServiceIdentity.Test/ScenarioTests/CrudTests.ps1]: There is a common cmdlet called Wit-Seconds that you shoudl use - however this should never be done without a test to ensure the condition you are watiting for has occurred.

File: src/ManagedServiceIdentity/ManagedServiceIdentity/Az.ManagedServiceIdentity.psd1 [hunk 4]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.7.1'
+ModuleVersion = '0.8.0'
```
Comment (markcowl) [inline src/ManagedServiceIdentity/ManagedServiceIdentity/Az.ManagedServiceIdentity.psd1]: Instead of updating the file, update the changelog

**Comment by cormacpayne** [comment]
@wendyNEU We're going to temporarily close this PR due to lack of activity for two weeks -- please feel free to re-open this PR when you're able to address the comments left above.

**Pull request closed without merging**